### PR TITLE
[Bugfix] Minor typo fix on Exercise Detail Page

### PR DESCRIPTION
**Pull request by lschlesinger**
### Checklist
- [ ] I tested *all* changes and *all* related features with different users (student, tutor, editor, instructor, admin) 
- [x] Client: I translated all newly inserted strings into English and German.
- [x] Language: I followed the [guidelines for inclusive, diversity-sensitive, and appreciative language](https://docs.artemis.ase.in.tum.de/dev/guidelines/language-guidelines/).

### Motivation and Context

Solves an issue that I found during testing: #4023 

### Steps for Testing

1. Log in to Artemis
2. Navigate to Exercise Detail page
3. Check for English and German if label for statistics is shown correctly

### Review Progress

- Code Review
  - [x] Review 1
  - [x] Review 2
- Manual Tests
  - [x] Test 1
  - [x] Test 2

### CODE REVIEW

File: src/main/webapp/i18n/de/statistics.json [hunk 1]
```diff
@@ -72,7 +72,7 @@
     },
     "exercise-statistics": {
         "title": "Aufgabenstatistiken",
-        "averageScore": "Duschnittspunktzahl",
+        "averageScore": "Durschnittspunktzahl",
```
Comment (jhainzlschmid) [inline src/main/webapp/i18n/de/statistics.json:75]: I small typo
```suggestion
        "averageScore": "Durchschnittspunktzahl",
```

**Review by jhainzlschmid — CHANGES_REQUESTED** [review]
Found a small typo. Didn't test it manually because I am fairly certain that it will work 😄

**Comment by lschlesinger** [comment]
@jhainzlschmid thanks, so stupid 🤦🏽‍♀️

**Review by jhainzlschmid — APPROVED** [review]
> @jhainzlschmid thanks, so stupid 🤦🏽‍♀️

Happens to the best of us^^
Typo has been fixed, I approve

**Review by Malyuk-A — APPROVED** [review]
Tested on TS2.

**Review by MerlinMehmed — APPROVED** [review]
Tested on TS2, the typos are fixed. Approve the code as well.

**Review by ndricimrr — APPROVED** [review]
Nice catch. Can confirm it worked locally as well 👍 :D

**Pull request merged**